tests/alembic/revisions/test_e9b705f9adf9_add_server_defaults_to_timestamp_.py

```
@@ -0,0 +1,284 @@
+        )
+
+        assert (
+            default == expected_default
+        ), f"Column {table}.{column} should have a server default of {expected_default} after migration, but default is {default}"
+
+    # Test that defaults actually work by inserting rows without explicit values
+    test_workspace_id = generate_nanoid()
+    conn.execute(
+        text(
+            f'INSERT INTO "{schema}"."workspaces" ("id", "name") '
+            + "VALUES (:id, :name)"
```

Ruff (0.14.2)

Explicitly concatenated string should be implicitly concatenated

Remove redundant '+' operator to implicitly concatenate

(ISC003)